python/mxnet/initializer.py

```diff
@@ -431,8 +432,30 @@ def _init_weight(self, _, arr):
 
 
 @register
+class LSTMCellForgetBias(Initializer):
+    """Initialize the bias of an LSTM to 0.0 except for the forget gate whose bias is set to custom value.
+
+    Parameters
+    ----------
+    num_hidden: int, number of hidden nodes in LSTMCell.
+
+    forget_bias: float, bias for the forget gate, Jozefowicz et al. 2015 recommends setting this to 1.0.
+    """
+    def __init__(self, num_hidden, forget_bias):
```

Comment:
I think it's safe to divide by 4 and omit the num_hidden arg here since we are assuming LSTM anyway